### PR TITLE
Update ansible scripts to work with newer versions of ansible

### PR DESCRIPTION
**Pull request by dexterlb**
This PR replaces usages of `include` by `{include,import}_tasks`. It has not been thoroughly tested, and I've decided between `include_tasks` and `import_tasks` by basically eyeballing it (please review carefully)

Thanks!

### CODE REVIEW

**Comment by krokodilerian** [comment]
Any chance to test this with 2.14?

File: ansible/hosts
Comment (aastefanov) [inline ansible/hosts:1]: We shouldn't need this, at least not in git

Comment (dexterlb) [reply ansible/hosts:1]: 👌🏿

**Comment by yoe** [comment]
> Any chance to test this with 2.14?

2.14 already issues deprecation warnings for the include statements. One might assume that the include_tasks/import_tasks features are available there then, too?

I would like to see this merged ASAP, as it's blocking me from configuring sreview properly (my laptop has 2.17, which does produce an error)

**Comment by aastefanov** [comment]
Note: We need to merge https://github.com/FOSDEM/ansible-bind/pull/3 as well

**Pull request merged**